dipy/workflows/segment.py

```diff
@@ -1,84 +1,85 @@
 from __future__ import division, print_function, absolute_import
 
-from glob import glob
-from os.path import join, basename, splitext
+import inspect
+import logging
+import numpy as np
 
 import nibabel as nib
-import numpy as np
 
-from dipy.workflows.utils import choose_create_out_dir
 from dipy.segment.mask import median_otsu
+from dipy.workflows.workflow import Workflow
+
+
+class MedianOtsuFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'medotsu'
+
+
+    def run(self, input_files, save_masked=False, median_radius=2, numpass=5,
+            autocrop=False, vol_idx=None, dilate=None, out_dir='',
+            out_mask='brain_mask.nii.gz', out_masked='dwi_masked.nii.gz'):
+        """ Workflow wrapping the median_otsu segmentation method.
+
+        Applies median_otsu segmentation on each file found by 'globing'
+        ``input_files`` and saves the results in a directory specified by
+        ``out_dir``.
+
+        For a quick brain segmentation, use --numpass 5 and --median_radius 2
+
+        Parameters
+        ----------
+        input_files : string
+            Path to the input volumes. This path may contain wildcards to
+            process multiple inputs at once.
+        save_masked : bool
+            Save mask
+        median_radius : int, optional
+            Radius (in voxels) of the applied median filter(default 2)
+        numpass : int, optional
+            Number of pass of the median filter (default 5)
```

Comment:
Its **passes** maybe? Or maybe you think it is not the right term. 
Its is the number of successive times the median filter is going to be applied on the image.